lib/features/add_property/view/select_property_type.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:hatspace/features/add_property/view/date_picker_view.dart';
+import 'package:hatspace/features/add_property/view/property_type_cart_view.dart';
+import 'package:hatspace/features/add_property/view_model/cubit/add_property_cubit.dart';
+import 'package:hatspace/features/add_property/view_model/cubit/property_type_cubit.dart';
+
+import 'package:hatspace/strings/l10n.dart';
+
+class SelectPropertyType extends StatelessWidget {
+  const SelectPropertyType({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<PropertyTypeCubit, PropertyTypeState>(
+        listener: (context, state) {
+      // Implement validate to enable next button
+      if (state.propertyTypes != null) {
+        context.read<AddPropertyCubit>().enableNextButton();
+      }
+    }, builder: (context, state) {
+      return Padding(
+        padding: const EdgeInsets.only(left: 16, top: 33, right: 16),
+        child: Wrap(
```

Comment:
We may have issue here if we run on tablets, since there will be much space on the screen, What kind of place and Choose kind of your property may be on the same line. Please check again on iPad simulator